### PR TITLE
Component demo enhancements

### LINKED ISSUE #860: [component demo] add repo links
Add links to component demo window 
- React: http://react.carbondesignsystem.com/
- Angular: http://angular.carbondesignsystem.com/
- Vanilla: ??
- Vue: http://vue.carbondesignsystem.com/

image

image

### Spec:

image

**enabled**
text color: `$inverse-01`
icon color: `$inverse-01`

**hover**
text color: `$inverse-01`
icon color: `$inverse-01`
underline: `$inverse-01`

**focus**
border: `inverse-focus-ui`

### PR DESCRIPTION
**Pull request by vpicone**
closes #860 

- fix: copy button cut off
- fix: use `IBM Plex Mono` instead of `Courier` in the code block
- feat: add links to CodeSandbox row

todo: ~replace fake links w/ proper links for frameworks~

### CODE REVIEW

**Comment by mjabbink** [comment]
At least Courier was also designed by IBM!

**Comment by vpicone** [comment]
@mjabbink what?!? That’s awesome. Gonna go on a Wikipedia binge.

**Comment by vpicone** [comment]
@aagonzales updated type treatment
@alisonjoseph I added a prop to define the links when using the mdx component. We can do a find and replace for `https://the-carbon-components.netlify.com/` whenever we get the subdomain setup. Considering it's vanilla I don't think it needs to be blocker just using the netlify url until then.

```jsx
<ComponentDemo
  links={{
    React:
      'http://react.carbondesignsystem.com/?path=/story/inlineloading--inline-loading',
    Angular:
      'https://angular.carbondesignsystem.com/?path=/story/components-inline-loading--basic',
    Vue:
      'http://vue.carbondesignsystem.com/?path=/story/components-cvinlineloading--default',
    Vanilla: 'https://the-carbon-components.netlify.com/?nav=inline-loading',
  }}
  knobs={{
    InlineLoading: ['status'],
  }}>{`<InlineLoading description="Loading..." />`}</ComponentDemo>
```

**Comment by alisonjoseph** [comment]
@vpicone looks good! was going to suggest the same thing, we can update that link later

File: src/components/ComponentDemo/Code/Code.module.scss [hunk 1]
```diff
@@ -44,15 +44,32 @@
...
+  }
+}
+
+.link-list a {
   display: flex;
   align-items: center;
   @include carbon--type-style('body-short-01');
   color: $inverse-01;
   text-decoration: none;
+  margin-right: 24px;
```
Comment (alisonjoseph) [inline src/components/ComponentDemo/Code/Code.module.scss]:
```suggestion
  margin-right:  $spacing-06;
```

File: src/components/ComponentDemo/ComponentDemo.module.scss [hunk 2]
```diff
@@ -172,6 +172,7 @@ fieldset.form-group {
 
 .editor-container pre,
 .editor-container textarea {
+  font-family: 'IBM Plex Mono' !important;
```
Comment (alisonjoseph) [inline src/components/ComponentDemo/ComponentDemo.module.scss]: Don't we have a token for the font family name? Or is that only in the theme? and do we need the important?

**Comment by vpicone** [comment]
@alisonjoseph react-live has some dumb inline styles for the code editor :(

there's ways to stop individual styles from making it through but I think given the scope it's just not worth figuring out haha.

**Comment by alisonjoseph** [comment]
worth maybe adding a comment in the code that its there to override the react live inline styles.

**Pull request merged**